airflow/example_dags/example_http_operator.py

```diff
@@ -40,55 +40,56 @@
 
 # t1, t2 and t3 are examples of tasks created by instantiating operators
 t1 = SimpleHttpOperator(
+    http_conn_id='http_for_example_dag',
     task_id='post_op',
-    endpoint='api/v1.0/nodes',
+    endpoint='/http_test',
     data=json.dumps({"priority": 5}),
     headers={"Content-Type": "application/json"},
-    response_check=lambda response: True if len(response.json()) == 0 else False,
+    response_check=lambda response: True if len(response.json()) > 0 else False,
     dag=dag)
 
 t5 = SimpleHttpOperator(
+    http_conn_id='http_for_example_dag',
     task_id='post_op_formenc',
-    endpoint='nodes/url',
+    endpoint='/http_test',
     data="name=Joe",
     headers={"Content-Type": "application/x-www-form-urlencoded"},
     dag=dag)
 
 t2 = SimpleHttpOperator(
+    http_conn_id='http_for_example_dag',
     task_id='get_op',
     method='GET',
-    endpoint='api/v1.0/nodes',
+    endpoint='/http_test',
     data={"param1": "value1", "param2": "value2"},
     headers={},
     dag=dag)
 
-t3 = SimpleHttpOperator(
-    task_id='put_op',
-    method='PUT',
-    endpoint='api/v1.0/nodes',
-    data=json.dumps({"priority": 5}),
-    headers={"Content-Type": "application/json"},
-    dag=dag)
+# t3 = SimpleHttpOperator(
```

Comment:
The data is not actually relevant to the test and the endpoints are to comply with REST principles as an example. I don't support DELETE or PUT in the demo, so these were commented. I can entirely remove them, because POST and GET were demonstrated. Theoretically though, DELETE and PUT are also supported, so just to show to a developer they could be used beyond the usual GET and POST.